### ISSUE TITLE
lagging

### ISSUE THREAD

**Issue by atari-lord** [Minecraft-Eternal/MC-Eternal-1.12#134]
Every 3 seconds my game lags for 5 seconds. Tried restarting and still happens.

MC Eternal 1.4.3.1

**Comment by criscky** [comment]
not enough information

What was in your inventory, equipments and baubles?
What are you doing?
Can you send a log and a debug from WHEN it happens
What args are you using?
What's your java version?
How can i reproduce this error?

Some simple things:

    please use args or else your gb will go nuts
    please be sure you're not using any fish's undead equipament or have it in your inventory (this is pinned in the discord server)
    please if you cannot provide the informations i asked, close this issue and seek help in our discord, this issue tracker is more for bugs in the pack for performance errors go to discord as stated in the message you get when you join a world.

**Comment by atari-lord** [comment]
Not sure why this issue got closed. 
As far as inventory I do not have anything from fish's undead, I've included a screenshot. As far as reproducing it, it just happens. I launch them game and it starts lagging. I tested it twice out by just hanging out(not really doing anything) and less than 10 minutes later started getting hit with lag spikes. Travelled to different areas and lag followed me everywhere. I attached the most recent log but don't know how to look up ARGS or JAVA version, minecraft version is 1.12.2
[latest.log](https://github.com/Minecraft-Eternal/MC-Eternal-1.12/files/6399593/latest.log)

![Screenshot (11)](https://user-images.githubusercontent.com/19884570/116559031-0117d300-a8ce-11eb-8dfc-07b296ed8b73.png)
